### DISCUSSION TITLE
Is this allowed

### DISCUSSION THREAD

**Discussion by FirephoenixX02** [maximmax42/Discord-CustomRP#176] [General]
Can you use this application without any problems or will you get banned from discord like for using self bots?

**Comment by maximmax42** [comment]
It's not a self-bot, it's like any other game with rich presence.

**Reply by FirephoenixX02** [reply]
> It's not a self-bot, it's like any other game with rich presence.

i know its not a selfbot but does it violate the discord tos?

**Reply by maximmax42** [reply]
It doesn't.
